### PR TITLE
Test: UnitTest for ZEROS

### LINKED ISSUE #1797: I want to  add the UnitTest for ZEROS
### Describe Current Status and Possible Solution

UnitTest for ZEROS is lacking

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by SeptemberMy**
Fix #1797

### CODE REVIEW

File: source/module_base/test/ylm_test.cpp [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+class ylmTest : public testing::Test
+{
+	
+};
+
+TEST_F(ylmTest,Zeros)
+{
+    double aaaa[100];
+    ModuleBase::Ylm::ZEROS(aaaa,100); 
+    EXPECT_EQ(aaaa[0],0.0);
```
Comment (hongriTianqi) [inline source/module_base/test/ylm_test.cpp]: Please delete repeated EXPECTs.

**Review by hongriTianqi — CHANGES_REQUESTED** [review]

File: source/module_base/test/ylm_test.cpp (hunk 1 above)
Comment (SeptemberMy) [reply source/module_base/test/ylm_test.cpp]: ok, I have already finished it. Please check it again.

File: source/module_base/test/ylm_test.cpp [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+class ylmTest : public testing::Test
+{
+	
+};
+
+TEST_F(ylmTest,Zeros)
+{
+    double aaaa[100];
+    ModuleBase::Ylm::ZEROS(aaaa,100); 
+   for(int i = 0; i < 100; i++)
```
Comment (hongriTianqi) [inline source/module_base/test/ylm_test.cpp]: Please make the indents consistent and delete blank line.

**Review by hongriTianqi — CHANGES_REQUESTED** [review]

File: source/module_base/test/ylm_test.cpp (hunk 2 above)
Comment (SeptemberMy) [reply source/module_base/test/ylm_test.cpp]: ok, I have already finished it.

**Review by hongriTianqi — APPROVED** [review]

File: source/module_base/test/ylm_test.cpp (hunk 2 above)
Comment (hongriTianqi) [reply source/module_base/test/ylm_test.cpp]: Thanks.

**Pull request merged**